.github/workflows/ci.yml

```diff
@@ -675,3 +677,24 @@ jobs:
           gh pr review ${{ github.event.pull_request.number }} --approve
           # Call the API directly to work around https://github.com/cli/cli/issues/8352
           gh api graphql -f query='mutation PullRequestAutoMerge {enablePullRequestAutoMerge(input: {pullRequestId: "${{ github.event.pull_request.node_id }}"}) {clientMutationId}}'
+  # This job will trigger another workflow such that it will trigger a re-run of this failing workflow
+  # We can't automatically do this here, since you can only re-run a workflow iff it has finished,
```

Comment:
```suggestion
  # We can't automatically do this here, since you can only re-run a workflow if it has finished,
```

Reply:
😅 "iff" is what we used in uni to mean "if and only iff", it wasn't a typo. let's merge this to avoid confusion though